app/views/va_facilities/_facility_map.html.erb

```diff
@@ -0,0 +1,29 @@
+<script>
+    // Initialize and add the map
+    function initMap() {
+        //default marker
+<!--        var pinImage = new google.maps.MarkerImage('<%= image_path('map-marker-selected.svg') %>',-->
+<!--            new google.maps.Size(29, 42),-->
+<!--            new google.maps.Point(0,0),-->
+<!--            new google.maps.Point(10, 34));-->
+
+        // The location of Uluru
+        const uluru = { lat: <%= @va_facility.latitude %>, lng: <%= @va_facility.longitude %> };
+        // The map, centered at Uluru
+        const map = new google.maps.Map(document.getElementById("map"), {
+            zoom: 8,
+            center: uluru,
+            //disableDefaultUI: true
+        });
+        // The marker, positioned at Uluru
+        const marker = new google.maps.Marker({
+            position: uluru,
+            map: map,
+            //icon: pinImage
+        });
+    }
+</script>
+<script
+  src="https://maps.googleapis.com/maps/api/js?key=<%= ENV['GOOGLE_API_KEY'] %>&callback=initMap&libraries=&v=weekly"
+  async
+></script>
```

Comment:
We should try to stay consistent with how we generate our maps. Use `home_map.es6` or `practice_map.es6` as reference.